vimPlugins followup

teto opened the pull request:
<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!
List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->

###### Motivation for this change

trying to address some vim issues possibly triggered by https://github.com/NixOS/nixpkgs/pull/136429



###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

Commit:
neovim.tests: complete plug test

to check that it can correctly load the colorscheme

r-rmcgibbo commented:
Result of `nixpkgs-review pr 137885` at b95ce9ce run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>8 packages built successfully:</summary>
  <ul>
    <li>spacevim</li>
    <li>tests.vim.test_vim_with_custom_plugin</li>
    <li>tests.vim.test_vim_with_vim_nix</li>
    <li>tests.vim.test_vim_with_vim_nix_using_pathogen</li>
    <li>tests.vim.test_vim_with_vim_nix_using_plug</li>
    <li>tests.vim.test_vim_with_vim_nix_using_vam</li>
    <li>tests.vim.vim_with_vim2nix</li>
    <li>vimacs</li>
  </ul>
</details>

figsoda commented:
@ofborg eval neovim.tests

Commit:
vimUtils: expose packDir

Commit:
vimUtils.vimGenDocHook: fix tag generation

Commit:
vimPlugins.minimap-vim: add test

SuperSandro2000 commented on pkgs/misc/vim-plugins/vim-utils.nix:
```diff
@@ -187,8 +187,10 @@ let
 
-  nativeImpl = packages:
-  (let
+  /* Generates a packpath folder as expected by vim
+  */
```
```suggestion
  # Generates a packpath folder as expected by vim
```

teto replied:
done

Commit:
neovim.tests: add a test for tags generation

packer.nvim has a doc/ folder so the vim doc hook should generate a 'tags' file

teto commented:
@minijackson could you check it fixes the tag generation for you ?

teto commented:
this does not fix the vim-plug issue (will try again once I can reproduce the issue), but it adds some more testing, especially for tag generation.

nixos-discourse commented:
This pull request has been mentioned on **NixOS Discourse**. There might be relevant details there:

https://discourse.nixos.org/t/nvim-helptags-not-generated/15083/2
